languages/es-ES.js

```diff
@@ -1,169 +1,169 @@
+			COMMAND_TRANSFER_ERROR: '❌ El archivo ha sido transferido o nunca existió.',
+			COMMAND_TRANSFER_SUCCESS: (type, name) => `✅ Transferida la pieza tipo ${type}: ${name} con éxito.`,
+			COMMAND_TRANSFER_FAILED: (type, name) => `La transferencia de la pieza tipo ${type}: ${name} al Cliente ha fallado. Por favor, revisa la consola.`,
+			COMMAND_RELOAD: (type, name) => `✅ Recargada la pieza tipo ${type}: ${name}`,
+			COMMAND_RELOAD_ALL: (type) => `✅ Recargadas todas las piezas tipo ${type}.`,
+			COMMAND_RELOAD_DESCRIPTION: 'Recarga una pieza de Klasa, o todas las piezas de una colección.',
+			COMMAND_REBOOT: 'Reiniciando...',
+			COMMAND_REBOOT_DESCRIPTION: 'Reinicia el bot.',
+			COMMAND_PING: '¿Ping?',
+			COMMAND_PING_DESCRIPTION: 'Ejecuta una prueba de conexión a Discord.',
+			COMMAND_PINGPONG: (diff, ping) => `¡Pong! (El viaje duró: ${diff}ms. Latencia: ${ping}ms.)`,
+			COMMAND_INVITE_SELFBOT: '¿Para qué quiere un enlace de invitación un selfbot?',
+			COMMAND_INVITE: (client) => [
+				`Para añadir ${client.user.username} a tu servidor de Discord:`,
+				`<${client.invite}>`,
+				util.codeBlock('', [
+					'El enlace de arriba está generado con los permisos necesarios para ejecutar todos los comandos actuales.',
+					'Entiendo que no todos los permisos son necesarios para todos los servidores, pero puedes cambiarlos sin problema.',
+					'Si intentas usar un comando que requira más permisos de los que tengo, te lo haré saber.'
+				].join(' ')),
+				'Por favor reporta a <https://github.com/dirigeants/klasa> si encuentras algún bug.'
+			],
+			COMMAND_INVITE_DESCRIPTION: 'Muestra el enlace de invitación para el bot.',
+			COMMAND_INFO: [
+				'Klasa es un framework \'plug-and-play\' construido encima de la librería Discord.js.',
+				'La mayor parte del código es modularizado, lo cual permite a los desarrolladores editar Klasa para encajar con sus necesidades.',
+				'',
+				'Algunas de las características de Klasa incluyen:',
+				'• 🐇💨 Carga muy rápida con soporte ES2017 (`async`/`await`).',
+				'• 🎚🎛 Configuración separada para cada servidor, usuario y cliente que puede ser expandida fácilmente.',
+				'• 💬 Un sistema de comandos personalizable con análisis automático de argumentos y fácil de usar, refrescar y desactivar módulos.',
+				'• 👀 "Monitores", los cuales pueden interactuar con todos los mensajes a modo de moderación (Filtro de palabras, protección de spam, etc).',
+				'• ⛔ "Inhibidores", los cuales pueden prevenir la ejecución de los comandos basándose en un set de parámetros (Permisos, Listas negras, etc).',
+				'• 🗄 "Proveedores", los cuales te permiten conectar Klasa con una base de datos cualquiera.',
+				'• ✅ "Finalizadores", los cuales son ejecutados después de un comando lanzado con éxito.',
+				'• ➕ "Extensibles", código que actua pasivamente. Añaden propiedades o métodos a las clases de Discord.js.',
+				'• 🌐 "Lenguajes", los cuales permiten usar diferentes lenguages en tu bot.',

```

Comment:
A typo for a typo! Fixed this 😅 

```suggestion
				'• 🌐 "Lenguajes", los cuales permiten usar diferentes lenguajes en tu bot.',
```